CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+CONTRIBUTING
+============
+
+We very much appreciate contributions in the form of pull requests (PRs).
+When preparing a PR here are some general guidelines:
+
+- Please read through and clean your code before making a PR. Clean
+  code has reasonable line length (<100 characters), good indentation,
```

Comment:
I know that 80 +- epsilon is also a common limit for line lengths. I have no really strong opinions about this as long as it's less than 100